shared/README.md

```diff
@@ -364,6 +364,22 @@ See [this
 link](https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers)
 for how to increase the watch limit; I set mine to 65536.
 
+On Android 28 and above HTTP traffic is disabled by default which can block
+Metro Bundler from running properly. We have manually allowed `127.0.0.1` to
+have HTTP traffic, so if you see an error about connecting to the bundler
+server you should manually change the dev server URL and then kill and restart
+the app:
+
+```sh
+# First enable loopback
+adb reverse tcp:8081 tcp:8081
+# Opens react native debug menu
+# https://facebook.github.io/react-native/docs/debugging.html#accessing-the-in-app-developer-menu
```

Comment:
you can just hit up volume

Reply:
ah good to know

Reply:
Since the linked page doesn't actually mention hitting up volume, may be good to mention this here too

Reply:
i'd rather leave as is and let their docs explain in case things change in the future